tools/run_tests/stress_test/configs/opt-tsan-asan.json

```diff
@@ -0,0 +1,135 @@
+{
```

Comment:
I think JSON config is fine (at least for now), but please note that we've been kind of pushing .yaml configurations within our repo.